packages/reactotron-server/src/server/reactotron/connectionHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { connections } from "../datastore"
+import { messaging, MessageTypes } from "../messaging"
+
+function onConnectionEstablished(connection) {
+  const newConnection = { clientId: connection.clientId, platform: connection.platform }
+  connections.addConnection(newConnection)
```

Comment:
`addConnection` does some logic that decides if we should actually add the connection or not. Not sure if we should skip the event on the next line if it doesn't. Right now I don't believe it will cause any harm